scripts/cut_release_branch.py

```diff
@@ -35,6 +35,20 @@
 
 
 def new_version_type(arg, pattern=re.compile(r'\d\.\d\.\d')):
+    r"""Checks that the new version name matches the expected pattern.
```

Comment:
Please remove this unnecessary 'r' before quotes.

Reply:
I need to use r before quotes because I'm using backslashes in the docstring. Ohterwise I get an error.

Reply:
What error are you getting? I think you should not get any as we can add anything in the triple quotes as the part of docstrings. Is there is explicit test from our Oppia's side for this?

Reply:
```
************* Module oppia.scripts.cut_release_branch
W: 38, 0: Anomalous backslash in string: '\d'. String constant might be missing an r prefix. (anomalous-backslash-in-string)
W: 38, 0: Anomalous backslash in string: '\.'. String constant might be missing an r prefix. (anomalous-backslash-in-string)
W: 38, 0: Anomalous backslash in string: '\d'. String constant might be missing an r prefix. (anomalous-backslash-in-string)
W: 38, 0: Anomalous backslash in string: '\.'. String constant might be missing an r prefix. (anomalous-backslash-in-string)
W: 38, 0: Anomalous backslash in string: '\d'. String constant might be missing an r prefix. (anomalous-backslash-in-string)
```